.changeset/yellow-goats-travel.md

```diff
@@ -2,4 +2,95 @@
 '@openfn/language-fhir': major
 ---
 
-`fhir` redesign
+### Added
```

Comment:
Oh dude, I'm sorry but this is way too much! I specifically said this didn't need to be a full migration guide.

Most of our release notes are quite concise.

I'll go through and make a few suggestions.